src/main/java/br/com/digituz/mailer/service/EmailService.java

```diff
@@ -19,13 +28,59 @@
 	@Autowired
 	private EmailRepository emailRepository;
 
+	@Autowired
+	private JavaMailSender javaMailSender;
+
+	@Value("${spring.mail.username}")
+	private String sender;
+
+	@Value("${mailer.replyTo}")
+	private String replyTo;
+
 	private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
 	public void saveEmails(Email email) {
 		this.emailRepository.save(email);
 		logger.info("Email saved");
 	}
 
+	public void sendEmails() throws MessagingException {
+
+		MimeMessage mimeMessage = javaMailSender.createMimeMessage();
+
+		MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, true);
+
+		emailsAll()
```

Comment:
Ola @brunokrebs , mesmo utilizando essa @Query : 	
`@Query("select  e from Email e join e.attachments join e.recipients where e.emailStatus in ?1")`
eu ainda continuo recebendo o 

`org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: br.com.digituz.mailer.model.Email.recipients, could not initialize proxy - no Session`